server-common/src/main/java/org/apache/cassandra/sidecar/common/server/data/CompactionStatsData.java

```diff
@@ -0,0 +1,296 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.server.data;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.cassandra.sidecar.common.DataObjectBuilder;
+
+/**
+ * Data object representing compaction statistics
+ */
+public class CompactionStatsData
+{
+    private final long concurrentCompactors;
+    private final Map<String, Map<String, Integer>> pendingTasks;
+    private final long totalPendingTasks;
+    private final long completedCompactions;
+    private final long dataCompacted;
+    private final long abortedCompactions;
+    private final long reducedCompactions;
+    private final long sstablesDroppedFromCompaction;
+    private final CompletedCompactionsRateData completedCompactionsRate;
+    private final List<ActiveCompactionEntryData> activeCompactions;
+    private final long activeCompactionsCount;
+    private final long activeCompactionsRemainingTime;
```

Comment:
nit: they can be just `public` and the getters can be removed. Not strong on this one. Feel free to ignore.